src/omotes_orchestrator/main.py

```diff
@@ -297,10 +302,20 @@ def new_job_submitted_handler(self, job_submission: JobSubmission) -> None:
             )
         else:
             logger.debug("New job %s was not yet registered. Registering and submitting.")
+
+            if not job_submission.HasField("timeout_ms"):
```

Comment:
I am very confused as to why these changes are showing. This was added in commit https://github.com/Project-OMOTES/orchestrator/commit/3e10ae836ffe4f763af73838672a3263a6728c29 on the 30th of July and this commit is NOT in the PR. It appears to be a Github bug. Perhaps merging main into this branch helps as I have seen this before and that usually fixes it.

Reply:
Yup, merging main into this branch fixed it again. Appears to be an issue where Github cannot figure out the merge apriory properly. Git used an ORT strategy so it wasn't just a simple fastforward.

Reply:
Thanks for checking this. I wasn't aware that git automatically shifts the target branch back to the main when the original target branch is deleted. A good tip!